base/show.jl

```diff
@@ -468,6 +468,20 @@ end
 show(io::IO, f::Function) = show_function(io, f, get(io, :compact, false)::Bool)
 print(io::IO, f::Function) = show_function(io, f, true)
 
+function show_function(io::IO, c::ComposedFunction, compact::Bool)
```

Comment:
I would put this method into operators.jl along with the other `ComposedFunction` methods.